articles/types-of-styling-in-css/index.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/types-of-styling-in-css/
+title: Types of CSS (Cascading Style Sheets)
+description: This article will go over the different styling options in CSS, how to apply them, and when best to implement each style.
+author:
+date: 2020-11-02T00:00:00-13:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/types-of-styling-in-css/hero.jpg
+    alt: Types of CSS (Cascading Style Sheets) example image
+
+---
+We will be going over Cascading Style Sheets commonly known as CSS. CSS is what turns your everyday markup language, HTML, into a one of a kind expression that leaves people in admiration. CSS complements the elements in HTML to help come up with various designs that make a website pop. From background colors, font-size, font-weight, font-family, and much more the design of a website becomes limited by only your imagination.
+<!--more-->
+CSS is the main playground for any developer. To use an analogy, CSS is like all the spices you add to your food to give it a rich, enticing aroma and the desired taste when you finally present it. However, just throwing things in there randomly doesn't guarantee greatness.
+
+You still have to know what goes with what, when to apply a specific style or when something just doesn't fit in. How you implement your CSS can either make or break your website. Below we'll be looking at the different styling options in CSS, how to apply them, and when best to implement each style.
+
+### Styling in CSS
+There are 3 distinct methods for styling in CSS. Each level of styling is given a different hierarchical priority and is used for different reasons.
+
+The 3 methods are further grouped into two categories. Namely *Internal CSS* and *External CSS*. *Internal CSS* is comprised of Local Styles and Page-Level Styles while *External CSS* is just as the name implies.
```

Comment:
This section is was a bit confusing to me - wasn't sure what are the 3 methods - and what are the two categories. 

the 2 categories are external and internal.

Then what are the 3 methods?

I had to scroll down to see the 3 style you are referring to were the page-level, local, and external styles. 

Links would help readers follow along and see the differences.